zaga-web/src/app/components/creatework-order/creatework-order.component.ts

```diff
@@ -85,233 +86,255 @@ export class CreateworkOrderComponent implements OnInit {
   selectedMachine;
   selectedMaterial;
 
-  nameFC : FormControl = new FormControl("");
-  
+  nameFC: FormControl = new FormControl("");
+
   filteredOptions: Observable<string[]>;
 
   ngOnInit() {
     this.query = this.nameFC.value;
```

Comment:
What is this used for?

Reply:
Deprecated, will delete in the next commit